src/runtime/virtcontainers/clh.go

```diff
@@ -457,6 +457,11 @@ func (clh *cloudHypervisor) CreateVM(ctx context.Context, id string, network Net
 		return err
 	}
 
+	if !(clh.config.HotPlugVFIO == config.RootPort || clh.config.HotPlugVFIO == config.NoPort) {
+		return fmt.Errorf("Unsupported hotplug vfio value: %s. Must be %s or %s",
+			clh.config.HotPlugVFIO, config.NoPort, config.RootPort)
+	}
+
```

Comment:
Since this is a configuration setting, there is `checkPCIeConfig` function that I added for the general use case. Do you think we should move this check there?

Reply:
yes, i had stumbled across this function but didn't think to extend it. done